### PR TITLE
Including gh pages changes

### PR DESCRIPTION
**Pull request by Subasri-Viswanathan**
Include gh-pages for helm deployment.

### CODE REVIEW

File: index.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+entries:
+  boldreports:
+  - apiVersion: v2
+    appVersion: 5.3.10
```
Comment (ranganathan-arumugam) [inline index.yaml:5]: Give current release version for app version and version of helm chart.

File: index.yaml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    created: "2023-12-16T17:26:20.3843848+05:30"
+    description: Make bolder business decisions with complete reporting solutions
+      and unleash the power of your data with paginated, printable reports and make
+      intelligent decisions.
+    digest: F06176FA9C35A366E962DF81993E33B6C101F27FC0E9860A010849824906EE07
+    home: https://www.boldreports.com/
+    icon: https://on-premise-demo.boldreports.com/reporting/images/application/boldreports/main_logo.svg
+    name: boldreports
```
Comment (ranganathan-arumugam) [inline index.yaml:13]: Give KC id for customer related helm chart name.

File: index.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    type: application
+    urls:
+    - https://boldreports.github.io/bold-reports-kubernetes/boldreports-5.3.10.tgz
```
Comment (ranganathan-arumugam) [inline index.yaml:16]: Give KC id for customer related helm chart files.

File: index.yaml (hunk 1 above)
Comment (Subasri-Viswanathan) [reply index.yaml]: Changed into current release version

File: index.yaml (hunk 3 above)
Comment (Subasri-Viswanathan) [reply index.yaml]: changed

**Comment by ranganathan-arumugam** [comment]
Changes are fine I'm approving this changes

**Review by Sankaralingam1667 — APPROVED** [review]

**Pull request merged**